### ISSUE TITLE
FutureWarning when using GeoPlots with GeoDataFrame

### ISSUE THREAD

**Issue by hitchhiker3010** [PatrikHlobil/Pandas-Bokeh#67]
Hi Patrick,

When using geoplots this future warning comes up.

> FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
>            projstring = _prepare_from_string(" ".join((projstring, projkwargs)))

I believe its happening from 
```
inProj = Proj(init="epsg:4326")
outProj = Proj(init="epsg:3857")
```
in geoplot.py code
Maybe just changing  it to "espg:4326" form should work.

**Comment by PatrikHlobil** [comment]
Hi @hitchhiker3010,

Thanks for reporting. I will fix this with the new release.

Best Patrik

**Comment by PatrikHlobil** [comment]
Fixes with release 0.5 of Pandas-Bokeh
